src/scores/continuous/standard_impl.py

```diff
@@ -418,3 +419,147 @@ def pbias(
+
+
+def kge(
+    fcst: xr.DataArray,
+    obs: xr.DataArray,
+    *,
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    scaling_factors: Optional[Union[list[float], np.ndarray]] = None,
+    return_components: bool = False,
+) -> XarrayLike:
+    # pylint: disable=too-many-locals
+    """
+    Calculate the Kling-Gupta Efficiency (KGE) between observed and simulated (or forecast) values.
+
+    KGE is a performance metric that decomposes the error into three components:
+    correlation, variability, and bias.
+    It is computed as:
+
+    .. math::
+        \\text{KGE} = 1 - \\sqrt{\\left[s_\\rho \\cdot (\\rho - 1)\\right]^2 +
+        \\left[s_\\alpha \\cdot (\\alpha - 1)\\right]^2 + \\left[s_\\beta \\cdot (\\beta - 1)\\right]^2}
+
+    .. math::
+        \\rho = \\frac{\\text{cov}(x, y)}{\\sigma_x \\sigma_y}
+
+    .. math::
+        \\alpha = \\frac{\\sigma_x}{\\sigma_y}
+
+    .. math::
+        \\beta = \\frac{\\mu_x}{\\mu_y}
+
+    where:
+        - :math:`\\rho`  = Pearson's correlation coefficient between observed and forecast values
+        - :math:`\\alpha` is the ratio of the standard deviations (variability ratio)
+        - :math:`\\beta` is the ratio of the means (bias)
+        - :math:`x` and :math:`y` are forecast and observed values, respectively
+        - :math:`\\mu_x` and :math:`\\mu_y` are the means of forecast and observed values, respectively
+        - :math:`\\sigma_x` and :math:`\\sigma_y` are the standard deviations of forecast and observed values, respectively
+        - :math:`s_\\rho`, :math:`s_\\alpha` and :math:`s_\\beta` are the scaling factors for the correlation coefficient :math:`\\rho`,
+            the variability term :math:`\\alpha` and the bias term :math:`\\beta`
+
+    Args:
+        fcst: Forecast or predicted variables.
+        obs: Observed variables.
+        reduce_dims: Optionally specify which dimensions to reduce when
+            calculating the KGE. All other dimensions will be preserved.
+        preserve_dims: Optionally specify which dimensions to preserve when
+            calculating the KGE. All other dimensions will be reduced. As a
+            special case, 'all' will allow all dimensions to be preserved. In
+            this case, the result will be in the same shape/dimensionality
+            as the forecast, and the errors will be the error at each
+            point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely.
+        scaling_factors : A 3-element vector or list describing the weights for each term in the KGE.
+            Defined by: scaling_factors = [s_rho, s_alpha, s_beta] to apply to the correlation (rho),
+            variability (alpha), and bias (beta) terms respectively. Defaults to (1.0, 1.0, 1.0). (*See
+            equation 10 in Gupta et al. (2009) for definitions of s_rho, s_alpha and s_beta*)
```

Comment:
```suggestion
            equation 10 in Gupta et al. (2009) for definitions of s_rho, s_alpha and s_beta*).
```
Adding a full stop for consistency with other dot points